autopush/src/client.rs

```diff
@@ -1344,6 +1344,10 @@ fn emit_metrics_for_send(metrics: &StatsdClient, notif: &Notification, source: &
     if notif.topic.is_some() {
         metrics.incr("ua.notification.topic").ok();
     }
+    metrics
+        .incr_with_tags("ua.notification.sent")
+        .with_tag("source", source)
+        .send();
```

Comment:
A thought: we could kill the `ua.notification.topic` above for a topic tag here like the other new metrics are doing